engine/src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateFactory.java

```diff
@@ -21,6 +21,7 @@ public class EmailTemplateFactory {
     private static final String RBAC = "rbac";
     private static final String SOURCES = "sources";
     private static final String VULNERABILITY = "vulnerability";
+    private static final String FLEET_MANAGEMENT = "fleet_management";
```

Comment:
If the information from https://issues.redhat.com/browse/NOTIF-502 is still relevant, the app name should be:
```suggestion
    private static final String FLEET_MANAGEMENT = "fleet-management";
```